packages/css/all.css

```
@@ -4077,17 +4077,17 @@
 }
 
 .seed-tabs__trigger:is(:selected, [data-selected]) {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-class selected 

See MDN web docs for more details.

(lint/correctness/noUnknownPseudoClass)